### ISSUE TITLE
libqt5core is missing from the image

### ISSUE THREAD

**Issue by shubham1637** [OpenMS/OpenMS#7092]
Hi,
    I have tried `ghcr.io/openms/openms-executables` version 3.0.0 and latest. It runs fine on my Macbook, however on the computing cluster I see this error
```
INFO:    Converting SIF file to temporary sandbox...
WARNING: underlay of /etc/localtime required more than 50 (92) bind mounts
OpenSwathWorkflow: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
```

I was wondering why is it so? I thought the docker image should contain libqt5core already?

**Comment by jpfeuffer** [comment]
Hi! Your host OS kernel on the cluster is too old for the Qt version used (and the way it is built by apt).
You can use the workaround in the following thread:

https://unix.stackexchange.com/questions/700290/existing-so-file-cannot-be-loaded-even-though-it-exists-seems-to-depend-on-doc

**Comment by shubham1637** [comment]
Thanks Julianus. However, when I am using version `2.8.0`, I do not see this issue. Most of the computing cluster use Linux 3.10, so `libQt5Core` issue would come up. Is it possible to fix it within the image?
On a side note I get this warning `WARNING: underlay of /etc/localtime required more than 50 (94) bind mounts`, do u know how to fix this?

**Comment by jpfeuffer** [comment]
The problem is that Qt 5.10+ is using newer system calls and therefore has a kernel version tag that is larger than your host system's.
Yes it can be fixed as shown in the link by forcefully stripping the kernel version tag but I would be hesitant to do that for the official container since it might have unknown implications.
Apparently the kernel version check was removed in newer glibc so a distribution upgrade to 23.04 might solve it as well.

**Comment by jpfeuffer** [comment]
You can ignore the warning.
It might be fixed by adding a mount point `/etc/localtime` to the container but I am not sure.

**Comment by jpfeuffer** [comment]
By the way, it seems like you are using singularity.
I would suggest to use `ghcr.io/openms/openms-executables-sif` to skip the lengthy and sometimes faulty auto conversion from the docker container.

**Comment by shubham1637** [comment]
Thanks,
   I tested with `openms-executables-sif:3.0.0`, I still get the same error regarding `libQt5Core.so.5: `. Hence, the error is not due to auto conversion from the docker container.

**Comment by shubham1637** [comment]
I also used `-B /etc/localtime:/etc/localtime` flag with the singularity image, but still see the same warning 
`WARNING: underlay of /etc/localtime required more than 50 (93) bind mounts`

**Comment by jpfeuffer** [comment]
Yes, the signularity image will not solve your problem (it will just improve your workflow a little). You need to follow the instructions in the linked SO thread.
`strip libQt* somekernelversiontag` inside the container.

Regarding mounting. Did you create the folder inside the container first? But I am not sure about this one.

**Comment by shubham1637** [comment]
I cannot create the folder inside, as I am using the singularity image. I am mounting `/etc/localtime`, but even then I am getting the warning on the cluster. However, I don't get this warning on my MacBook.
